examples/cfg/example/intrinsics.yml

```diff
@@ -0,0 +1,13 @@
+---
+intrinsics:
+  intrinsics:
```

Comment:
@thomasgoodfellow What is the reason for the nested `intrinsics.intrinsics` here?

Reply:
That's regrettably ugly, emerging from my (mis?)use of dacite to have an optional list of intrinsic definitions. I'd certainly like to remove one layer of that onion but the formulations I tried didn't parse the YML I fed them

Reply:
I would propose to move the intrinsics definitions under `extensions:` as the intrinstics are usually set specific. But this can be done in a followup-PR

Reply:
Makes sense. BTW are you back earlier than scheduled, or not really back?

Reply:
I will be officially back on Tuesday. But I got some time to work on my personal coding projects and while I am waiting for docker images to build, I can do some Seal5-related work ;)